### DISCUSSION TITLE
Do you want the calculator files to be in a different folder?

### DISCUSSION THREAD

**Discussion by JohnLSison** [srothetafesa/Tafe24S1T109_Ext#5] [General]
Hey Sam,

Just wondering, before I push any changes to my feature branch, do you want me to put the new files in it's own folder or are you fine with them being placed in the main folder? 

Cheers,
John

**Comment by srothetafesa** [comment]
Hi John,

I think keep all files where they are, so when we merge there will be no issues.

Thanks,
Sam
